ui/src/components/ConnectorForm/index.tsx

```
@@ -18,50 +18,60 @@ export interface ConnectorFormRef {
+//                                 }}
+//                             />
+//                         ))}
+//                     </form>
+//                 </Form>
+//             </div>
+//         );
+//     }
+// );
+// export default ConnectorForm;
+
+const ConnectorForm: React.FC<ConnectorFormProps> = ({}) => {};
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)